src/main/java/seedu/address/MainApp.java

```diff
@@ -79,7 +83,8 @@ public void init() throws Exception {
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         Optional<ReadOnlyPersonBook> personModelOptional;
         ReadOnlyPersonBook personModel;
```

Comment:
Consider renaming them to personBookOptional and personBook to keep it consistent